src/main/java/com/lewisd/maven/lint/rules/AbstractReduntantVersionRule.java

```diff
@@ -47,20 +47,20 @@ protected void checkForRedundantVersions(final MavenProject mavenProject,
         }
 
         String version = resolveVersion(modelObject, resolvedModelObject);
-        String inheritedVersion = modelUtil.getVersion(tryResolveObject(inheritedObject));
+        String inheritedVersion = getModelUtil().getVersion(tryResolveObject(inheritedObject));

```

Comment:
It's kind of funny, I had this a one point in an older version, then switched to exposing it as a `protected final` field.  I usually agree with using getters, but in some cases, particularly for "utility" classes like `modelUtil`, I find it cleaner not to.
As I was writing this comment, I had a sense of deja-vu. :) https://github.com/lewisd32/lint-maven-plugin/pull/5#issuecomment-23230847  We've had this conversation before, lol.